packages/ppx/src/declarations_to_string.re

```diff
@@ -135,9 +138,22 @@ let rec render_function_calc = calc_sum => {
     let op = pick_operation(List.hd(list_of_sums));
     let first = render_product(product);
     let second = render_list_of_sums(list_of_sums);
-    [%expr "calc(" + [%e first] ++ [%e op] ++ [%e second] ++ ")"];
+    [%expr "calc(" ++ [%e first] ++ [%e op] ++ [%e second] ++ ")"];
+  }
+and render_function_min_or_max = calc_sums => {
+  switch (calc_sums) {
+  | [] => failwith("expected at least one argument")
```

Comment:
failwith is a no-no in code-gen (I know there are a few, but should be removed as well)

Create a new exception like: Wrong_value (if it's case of a parser error or Impossible in case of imposible parsing state)

Reply:
This should raise, never failwith